src/styles/blocks/container.scss

```diff
@@ -0,0 +1,23 @@
+.container {
+  width: $container-width;
+
+  .calendar {
```

Comment:
create a separate file for bem blocks (checklist)
<img width="647" alt="image" src="https://user-images.githubusercontent.com/102411130/214516690-f5e87317-bf16-4181-835c-506c16c16da4.png">